frontend/ui/widget/infomessage.lua

```diff
@@ -77,6 +78,10 @@ local InfoMessage = InputContainer:extend{
 }
 
 function InfoMessage:init()
+    if not self.font_adjusted then
+        self.face = Font:getFace(self.monospace_font and "infont" or "infofont")
```

Comment:
Or just `if not self.face then self.face = ...` ?
(Will allow passing a `face=` when we want something other of infont/infofont?)